tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -37,6 +37,7 @@
 
 metadata_1_versions = [str(LATEST_0_10_0)]
 metadata_2_versions = [str(LATEST_0_10_1), str(LATEST_0_10_2), str(LATEST_0_11_0), str(LATEST_1_0), str(LATEST_1_1)]
+fk_join_versions = [str(V_2_4_0)]
```

Comment:
We only test with latest bug-fix releases, so we can use `LATEST_2_4` here.

Reply:
Should we test all versions 2.4 - 3.1 ?

Reply:
Done in [d530a29](https://github.com/apache/kafka/pull/12122/commits/d530a29d1df6bb5302745c54ef08c83dc6c9c6de).